hr_employee_cost_history/models/hr_employee_timesheet_cost_history.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeeTimesheetCostHistory(models.Model):
+    _name = "hr.employee.timesheet.cost.history"
+    _description = "Employee Timesheet Cost History"
+    _order = "create_date DESC"
+
+    employee_id = fields.Many2one(
+        comodel_name="hr.employee",
+        string="Employee",
+    )
+    currency_id = fields.Many2one(
+        comodel_name="res.currency",
+        string="Currency",
+    )
+    timesheet_cost = fields.Monetary(currency_field="currency_id")
+    starting_date = fields.Date(
+        help="Change de timesheet cost from this date to now.",
```

Comment:
issue: you have a typo, but actually I think this can be reworded a bit to make it easier to understand:

```suggestion
        help="The cost change has effect since this date.",
```

Reply:
Changes applied and translations updated. Thanks @yajo !